### PR TITLE
Add self introduction about Jeungmin

### PR DESCRIPTION
**Pull request by ohtangza**
From the reference of React Native repository, I created my introduction.

### CODE REVIEW

File: JeungminOh/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# About Jeungmin Oh
```
Comment (ohtangza) [inline JeungminOh/README.md]: 잘하시네요...

File: JeungminOh/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- Hobby: playing piano, programming
```
Comment (yumi2011) [inline JeungminOh/README.md]: Where is weight training for your hobby!!!

Comment (ghsdh3409) [inline JeungminOh/README.md]: Wow! Piano!

Comment (ghsdh3409) [inline JeungminOh/README.md]: ???

Comment (Hayoung-jung) [inline JeungminOh/README.md]: 오 👍

Comment (ohtangza) [inline JeungminOh/README.md]: 똑바로 하세요.

Comment (ohtangza) [inline JeungminOh/README.md]: 왜 가독성이 안좋나요?

Comment (ohtangza) [inline JeungminOh/README.md]: 오 따봉썼다.

**Pull request merged**